gcloud-java-contrib/gcloud-java-nio/src/test/java/com/google/gcloud/storage/contrib/nio/CloudStorageOptionsTest.java

```diff
@@ -80,9 +87,12 @@ public void testWithUserMetadata() throws Exception {
     Files.write(path, "(✿◕ ‿◕ )ノ".getBytes(UTF_8),
         withUserMetadata("nolo", "contendere"),
         withUserMetadata("eternal", "sadness"));
-    GcsFileMetadata metadata = getMetadata("bucket", "path");
-    assertThat(metadata.getOptions().getUserMetadata().get("nolo")).isEqualTo("contendere");
-    assertThat(metadata.getOptions().getUserMetadata().get("eternal")).isEqualTo("sadness");
+    assertThat(
+        Files.readAttributes(path, CloudStorageFileAttributes.class).userMetadata().get("nolo"))
```

Comment:
Additional four spaces of indent here, and you've just barely made 100 characters. This is because `Files.read...` is not "syntactically parallel" with `.isEqualTo...` according to the style guide.